packages/platform/src/PlatformConfigProvider.ts

```diff
@@ -111,3 +112,32 @@ export const layerFileTree = (options?: {
     Effect.map(Layer.setConfigProvider),
     Layer.unwrapEffect
   )
+
+/**
+ * Create a dotenv ConfigProvider.
+ *
+ * @category constructors
+ * @since 1.0.0
+ */
+export const fromDotEnv: (
+  paths: string
+) => Effect.Effect<ConfigProvider.ConfigProvider, PlatformError, FileSystem.FileSystem> = internal.fromDotEnv
+
+/**
+ * Add the dotenv ConfigProvider to the environment, as a fallback to the current ConfigProvider.
+ * If the file is not found, a debug log is produced and empty layer is returned.
```

Comment:
This statement doesn't seem to be correct atm.

Reply:
Ouch, I removed the behaviour, nice catch. What do you think is better
- the current one, fail if the file doesn't exist
- the one described in the docstring

Reply:
I think using .env only for dev is quite common, so a debug log + empty layer sounds good. You could add a `ensureFileExists` option to toggle the behaviour.

Reply:
Agree, debug log it is. I think we don't need to make it configurable, this really feels like the most expected behaviour.